androidApp/src/main/res/raw/sample_assessment.json

```diff
@@ -84,6 +84,28 @@
       "optional": false,
       "inputItem":{"type" : "year", "placeholder" :  "Enter year"},
       "skipCheckbox":{"type":"skipCheckbox","fieldLabel":"No answer"}
+    },
+    {
+      "identifier": "multipleInputQuestion",
+      "type": "multipleInputQuestion",
+      "title": "Multiple Questions",
+      "subtitle": "Subtitle goes here",
+      "inputItems": [{"type" : "string", "placeholder" :  "Enter text here"},
+        {"type" : "string", "placeholder" :  "Enter text here"},
+        {"type" : "string", "placeholder" :  "Enter text here"},
+        {"type" : "string", "placeholder" :  "Enter text here"},
+        {"type" : "string", "placeholder" :  "Enter text here"}
+      ],
```

Comment:
Not important, but could make this a more concrete example such as: What is your name? with First Name and Last Name inputItems.